From b9ec19c29944411d5f530614bb23b5325df4fdb2 Mon Sep 17 00:00:00 2001
From: Pankit Garg <pankit.garg@nxp.com>
Date: Mon, 6 Jul 2020 13:43:32 +0530
Subject: [PATCH 07/22] dt-bindings: arm: fsl: Add device tree binding for
 ls1046a-db board

Add bindings for the ls1046adb board based on ls1046a SoC

Signed-off-by: Pankit Garg <pankit.garg@nxp.com>
---
 Documentation/devicetree/bindings/arm/fsl.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/fsl.yaml b/Documentation/devicetree/bindings/arm/fsl.yaml
index 32910306fa24..89e36c22467a 100644
--- a/Documentation/devicetree/bindings/arm/fsl.yaml
+++ b/Documentation/devicetree/bindings/arm/fsl.yaml
@@ -309,6 +309,7 @@ properties:
       - description: LS1046A based Boards
         items:
           - enum:
+              - fsl,ls1046a-db
               - fsl,ls1046a-wrdb
               - fsl,ls1046a-frwy
               - fsl,ls1046a-qds
-- 
2.17.1

